src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```

Comment:
As I said previously, let's do this change to `HANGUL_CHARSET`
I just posted my findings but didn't want you to remove them from the PR.